insalan/cms/models.py

```diff
@@ -78,6 +79,25 @@ def save(self, *args, **kwargs):
                 "border-right:1px SOLID #000000",
                 self.content
             )
+
+            letters = string.ascii_uppercase
```

Comment:
This whole code is uncommented and I had to do a non-trivial amount of work to try and understand why it's here. In my opinion, there should be no obscure automated processing of contents in the backend, at least none that requires you to go check out the *source code*. In five years someone will try and get why the things they are writing in the plannings are broken, and they will spend time trying to find where this processing is done. At least add comments for a rationale for this.

Reply:
Yes, it's mb for that, i should have added comments.

Reply:
You can comment even more more globally on this part. We don't have any idea what the expected input or output should look like (we can't event start gsheet because we don't know what the sheet should look like). We can't tell you the code does its work if we don't know what its work is supposed to be.

Reply:
I can but this will overload the codebase, I think it's better to add an article about this on the wiki and maybe link the code to it.